release.sh

```diff
@@ -80,3 +88,4 @@ case $1 in
         ;;
 esac
 
+trap - EXIT
```

Comment:
I didn't know you could do this. ^_^

Reply:
So `trap function EXIT` means "execute function on receiving the EXIT signal", which includes normal exit and errors. (I'm not sure what it does on `exec` 🤔) `trap - EXIT` means "clear the handler for the EXIT signal".

This works for trapping real signals too (`EXIT` is kind of a special, synthetic signal Bash adds), but I've not really found a use for that yet. Trapping `EXIT`, however, is really useful in lots of situations.

Reply:
Yeah, I knew you could write a `trap`; I didn't know you could remove it again. 🙂

Reply:
Note that (IIUC) you're clearing all handlers for that signal; I don't think you can have more than one, but you have to be a bit careful that you may not be removing the one you think you have.